projects/gnome.org/PyGObject/package.yml

```diff
@@ -0,0 +1,40 @@
+distributable:
+  url: https://download.gnome.org/sources/pygobject/{{version.marketing}}/pygobject-{{version}}.tar.xz
+  strip-components: 1
+versions:
+  url: https://download.gnome.org/sources/pygobject/cache.json
+  match: /pygobject-\d+\.\d+\.\d+\.tar\.xz/
+  strip:
+    - /^pygobject-/
+    - /\.tar\.xz/
+dependencies:
+  gnome.org/gobject-introspection: '*'
+  cairographics.org/pycairo: '*'
+runtime:
+  env:
+    PYTHONPATH: "{{prefix}}/lib/python3.11/site-packages:$PYTHONPATH"
```

Comment:
I think using a specific minor could be dangerous in the future. We should be doing:
```yaml
- run: ln -s python{{deps.python.org.version.marketing}} python{{deps.python.org.version.major}}
  working-directory: ${{prefix}}/lib
```
so we can use `version.major` in these envvar.